### DISCUSSION TITLE
在tabs中实现左右箭头切换tab头的功能，找了一圈没有找到，看到有网友说可以用renderTabBar来实现，但是renderTabBar没有一个具体的例子来说明这个api的强大功能

### DISCUSSION THREAD

**Discussion by yuehongyuan** [ant-design/ant-design#31213] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
解决tabs左右箭头切换tab头的问题

### What does the proposed API look like?
期望tabs左右箭头切换tab头的问题

**Comment by KiritoXF** [comment]
感觉可以监听键盘事件来实现
```
onKeyDown={((event) => {
  if (event.code === 'ArrowLeft') {
    // do sth.
  }
  if (event.code === 'ArrowRight') {
    // do sth.
  }
})}
```
然后设个变量记录 activeKey，按下按键后就变更值，再判断下到头了该怎么办

**Comment by muyiyang1** [comment]
哥们有方案了吗？ 我们也要做这个功能。。
